tests/testthat/test-Copula.R

```diff
@@ -73,14 +73,14 @@ test_that("Copula density is same in VineCopula and TMB", {
 
 test_that("Copula cdf is same in VineCopula and TMB", {
   nreps <- 20
-  test_descr <- expand.grid(family = c(3, 4, 5), # add copula families
+  test_descr <- expand.grid(family = c(1, 3, 4, 5), # add copula families
```

Comment:
It might be clearer to not include family `1` here until `pgaussian()` does what it's supposed to.

Also, the PDF test is missing.